Sokoliuk/HW6/HW6_task_01.py

```diff
@@ -0,0 +1,22 @@
+numbers = list(range(1, 11))
+ls_even, ls_odd, ls_other = [], [], []
+
+ls_even = [number for number in numbers if number % 2 == 0]
+print("Even numbers that divisible by 2: ", ls_even)
+
+ls_odd = [number for number in numbers if number % 3 == 0]
```

Comment:
your output will be [3, 6, 9] but 6 is not odd number
this implementation is wrong